src/utils/readtime.utils.js

```diff
@@ -0,0 +1,72 @@
+import moment from 'moment';
+import momentDurationFormatSetup from 'moment-duration-format';
+import faker from 'faker';
+
+momentDurationFormatSetup(moment);
+
+/**
+ * @description counts the number of words in a string.
+ * excluding spaces, tab and newlines
+ * @param {string} words
+ * @returns {number} number of words
+ */
+export function wordCount(words) {
+  return Math.floor(words.split(/[^\s]+/).length - 1);
+}
+
+/**
+ * @description calculate the time to view images in an article
+ * @param {array} images
+ * @returns {number} time in seconds for all images
+ */
+export function calcImageTime(images) {
+  let initSeconds = 11;
```

Comment:
The variable `initSeconds` can be more descriptive, I think. From what I see here, it is supposed to be the time it takes to view an image.

Reply:
thank, am on it.